integrations/deploy.yaml

```diff
@@ -43,8 +43,228 @@
+    - docker
+    - container
+    - containers
+  description: |
+    Install and connect new Docker containers
+    Find the commands for `docker run`, `docker compose` or `Docker Swarm`. On the last two you can copy the configs, then run `docker-compose up -d` in the same directory as the `docker-compose.yml`
+    ```
+    Netdata container requires different privileges and mounts to provide functionality similar to that provided by Netdata installed on the host. More info [here](https://learn.netdata.cloud/docs/installing/docker?_gl=1*f2xcnf*_ga*MTI1MTUwMzU0OS4xNjg2NjM1MDA1*_ga_J69Z2JCTFB*MTY5MDMxMDIyMS40MS4xLjE2OTAzMTAzNjkuNTguMC4w#create-a-new-netdata-agent-container)
+    Netdata will use the hostname from the container in which it is run instead of that of the host system. To change the default hostname check [here](https://learn.netdata.cloud/docs/agent/packaging/docker?_gl=1*i5weve*_ga*MTI1MTUwMzU0OS4xNjg2NjM1MDA1*_ga_J69Z2JCTFB*MTY5MDMxMjM4Ny40Mi4xLjE2OTAzMTIzOTAuNTcuMC4w#change-the-default-hostname)
+    ```
+  methods:
+    - method: Docker CLI
+      commands:
+        - channel: nightly
+          command: |
+            docker run -d --name=netdata \
+            --pid=host \
+            --network=host \
+            -v netdataconfig:/etc/netdata \
+            -v netdatalib:/var/lib/netdata \
+            -v netdatacache:/var/cache/netdata \
+            -v /etc/passwd:/host/etc/passwd:ro \
+            -v /etc/group:/host/etc/group:ro \
+            -v /proc:/host/proc:ro \
+            -v /sys:/host/sys:ro \
+            -v /etc/os-release:/host/etc/os-release:ro \
+            -v /var/run/docker.sock:/var/run/docker.sock:ro \
+            --restart unless-stopped \
+            --cap-add SYS_PTRACE \
+            --cap-add SYS_ADMIN \
+            --security-opt apparmor=unconfined \
+            netdata/netdata:edge \
+            -e NETDATA_CLAIM_TOKEN= {% claim_token %} \
+            -e NETDATA_CLAIM_URL={% claim_url %}
+        - channel: stable
+          command: |
+            docker run -d --name=netdata \
+            --pid=host \
+            --network=host \
+            -v netdataconfig:/etc/netdata \
+            -v netdatalib:/var/lib/netdata \
+            -v netdatacache:/var/cache/netdata \
+            -v /etc/passwd:/host/etc/passwd:ro \
+            -v /etc/group:/host/etc/group:ro \
+            -v /proc:/host/proc:ro \
+            -v /sys:/host/sys:ro \
+            -v /etc/os-release:/host/etc/os-release:ro \
+            -v /var/run/docker.sock:/var/run/docker.sock:ro \
+            --restart unless-stopped \
+            --cap-add SYS_PTRACE \
+            --cap-add SYS_ADMIN \
+            --security-opt apparmor=unconfined \
+            netdata/netdata:stable \
+            -e NETDATA_CLAIM_TOKEN= {% claim_token %} \
+            -e NETDATA_CLAIM_URL={% claim_url %}
+  additional_info: ''
+  related_resources: {}
+  platform_info:
+    group: 'no_include'
+    distro: 'docker'
+  quick_start: 1
```

Comment:
What about Windows?

Reply:
@shyamvalsan : I will need your help in updating Windows.. I have now updated all the others.

Reply:
```
1. Install [Windows Exporter](https://github.com/prometheus-community/windows_exporter) on every Windows host you want to monitor.
2. Install Netdata agent on Linux, FreeBSD or Mac.
3. Configure Netdata to collect data remotely from your Windows hosts by adding one job per host to windows.conf file. See the [configuration section](https://learn.netdata.cloud/docs/data-collection/monitor-anything/System%20Metrics/Windows-machines#configuration) for details.
4. Enable virtual nodes configuration so the windows nodes are displayed as separate nodes.
```